### PR TITLE
libiptcdata, fix build for x86_64, add defineDebugInfoPackage

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: media-libs/libiptcdata/libiptcdata-1.0.4.recipe [hunk 1]
```diff
@@ -8,18 +8,21 @@ this metadata."
 HOMEPAGE="http://libiptcdata.sourceforge.net/"
 COPYRIGHT="2005 David Moore"
 LICENSE="GNU LGPL v2.1"
-REVISION="2"
+REVISION="3"
 SOURCE_URI="http://prdownloads.sourceforge.net/libiptcdata/libiptcdata-1.0.4.tar.gz?download"
```
Comment (fbrosson) [inline media-libs/libiptcdata/libiptcdata-1.0.4.recipe]:
```diff
-SOURCE_URI="http://prdownloads.sourceforge.net/libiptcdata/libiptcdata-1.0.4.tar.gz?download"
+SOURCE_URI="https://downloads.sourceforge.net/libiptcdata/libiptcdata-$portVersion.tar.gz"
```

File: media-libs/libiptcdata/libiptcdata-1.0.4.recipe [hunk 2]
```diff
@@ -8,18 +8,21 @@ this metadata."
...
 CHECKSUM_SHA256="79f63b8ce71ee45cefd34efbb66e39a22101443f4060809b8fc29c5eebdcee0e"
 SOURCE_FILENAME="libiptcdata-1.0.4.tar.gz"
 PATCHES="libiptcdata-1.0.4.patch"
```
Comment (fbrosson) [inline media-libs/libiptcdata/libiptcdata-1.0.4.recipe]:
```diff
-SOURCE_FILENAME="libiptcdata-1.0.4.tar.gz"
+SOURCE_FILENAME="libiptcdata-$portVersion.tar.gz"
-PATCHES="libiptcdata-1.0.4.patch"
+PATCHES="libiptcdata-$portVersion.patch"
```

File: media-libs/libiptcdata/libiptcdata-1.0.4.recipe [hunk 3]
```diff
@@ -40,30 +43,33 @@ BUILD_REQUIRES="
 	devel:libiconv
 	"
 BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autopoint
 	cmd:gcc
+	cmd:libtool
```
Comment (fbrosson) [inline media-libs/libiptcdata/libiptcdata-1.0.4.recipe]:
```diff
-	cmd:libtool
+	cmd:libtoolize
```

File: media-libs/libiptcdata/libiptcdata-1.0.4.recipe (hunk 1 above)
Comment (Begasus) [reply media-libs/libiptcdata/libiptcdata-1.0.4.recipe]: Oh bugger, didn't capture these ones :)

**Pull request merged**